src/parser.ts

```diff
@@ -87,6 +87,9 @@ export const EXECUTION_TYPES: Record<StatementType, ExecutionType> = {
   ALTER_FUNCTION: 'MODIFICATION',
   ALTER_INDEX: 'MODIFICATION',
   ALTER_PROCEDURE: 'MODIFICATION',
+  BEGIN_TRANSACTION: 'MODIFICATION',
+  COMMIT: 'MODIFICATION',
+  ROLLBACK: 'MODIFICATION',
```

Comment:
I think that we should add a new `ExecutionType` named `TRANSACTION` here instead of using `MODIFICATION` as these statements aren't really modifying anything (and you can have just select queries inside a transaction).